tool/tctl/common/collection.go

```diff
@@ -1024,3 +1026,24 @@ func (c *samlIDPServiceProviderCollection) writeText(w io.Writer) error {
 	_, err := t.AsBuffer().WriteTo(w)
 	return trace.Wrap(err)
 }
+
+type deviceCollection struct {
+	devices []*devicepb.Device
+}
+
+func (c *deviceCollection) resources() []types.Resource {
+	r := make([]types.Resource, len(c.devices))
+	for i, resource := range c.devices {
+		r[i] = device.ProtoToResource(resource)
+	}
+	return r
+}
+
+func (c *deviceCollection) writeText(w io.Writer) error {
+	t := asciitable.MakeTable([]string{"ID", "OS Type", "Asset Tag", "Enrollment Status", "Creation Time", "Last Updated"})
+	for _, device := range c.devices {
+		t.AddRow([]string{device.Id, device.OsType.String(), device.AssetTag, devicepb.DeviceEnrollStatus_name[int32(device.GetEnrollStatus())], device.CreateTime.AsTime().Format(time.RFC3339), device.UpdateTime.AsTime().Format(time.RFC3339)})
```

Comment:
Suggestion: line breaks for improved readability, use "friendly" methods from lib/devicetrust:

```suggestion
		t.AddRow([]string{
			device.Id,
			devicetrust.FriendlyOSType(device.OsType),
			device.AssetTag,
			devicetrust.FriendlyDeviceEnrollStatus(device.EnrollStatus),
			device.CreateTime.AsTime().Format(time.RFC3339),
			device.UpdateTime.AsTime().Format(time.RFC3339),
		})
```